### DISCUSSION TITLE
记录一下配置Clash透明代理 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#22] [General]
# 记录一下配置Clash透明代理 | 倾旋的博客

日常办公和上网需要挂不同的代理，切换起来有点麻烦，所以记录一下配置Clash透明代理的流水账

https://payloads.online/article/74116aab-0b33-4644-a6cc-28de0b36214d

**Comment by heheda123123** [comment]
v2raya不需要任何配置就支持透明代理

**Reply by Rvn0xsy** [reply]
棒！

**Comment by coutcin-xw** [comment]
师傅，我目前也是Archlinux作为主系统在使用，看您使用了Clash作为代理软件，但是对于日常的访问google等需求，纯Clash比较难用，使用clash-for-windows可以吗

**Comment by Rvn0xsy** [comment]
Clash for Windows不就是包装了一个图形界面吗。

我直接通过浏览器管理Clash的，一样的效果。

你可以在Github搜索这个项目：yacd ，或者 Clash Dashboard，支持Docker部署。



 ------------------原始邮件 ------------------
发件人：CoutCin ***@***.***>
日期：2023-08-13 下午7:58:07
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 记录一下配置Clash透明代理 | 倾旋的博客 (Discussion #22)
师傅，我目前也是Archlinux作为主系统在使用，看您使用了Clash作为代理软件，但是对于日常的访问google等需求，纯Clash比较难用，使用clash-for-windows可以吗

**Comment by xxs-2** [comment]
这不上openwrt+openclash

**Comment by Rvn0xsy** [comment]
😁在公司上班，用的台式机，家里倒是可以这么整



 ------------------原始邮件 ------------------
发件人：xxs ***@***.***>
日期：2023-08-15 下午4:00:10
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 记录一下配置Clash透明代理 | 倾旋的博客 (Discussion #22)
这不上openwrt+openclash
